test/jdk/java/awt/Frame/MultiScreenTest.java

```diff
@@ -173,7 +175,7 @@ public void paint( Graphics _g ) {
         g.setColor(Color.cyan);
         g.fillArc(150, 30, 30, 30, 0, 200);
     }
-
+@Override
```

Comment:
You may add a blank line before `@Override` line to maintain consistency. Also add 4 spaces to move `@Override` similar to other methods
```suggestion
    @Override
```